### ISSUE TITLE
composer install error( iisoft/yii2-composer 2.0.0 -> satisfiable by yiisoft/yii2-composer[2.0.0]. )

### ISSUE THREAD

**Issue by puppyLyx** [yiisoft/yii2#13931]
composer install
```
The "extra.asset-installer-paths" option is deprecated, use the "config.fxp-asset.installer-paths" option
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for yiisoft/yii2-composer 2.0.0 -> satisfiable by yiisoft/yii2-composer[2.0.0].
    - yiisoft/yii2-composer 2.0.0 requires composer-plugin-api 1.0.0 -> no matching package found.
  Problem 2
    - yiisoft/yii2-composer 2.0.0 requires composer-plugin-api 1.0.0 -> no matching package found.
    - yiisoft/yii2 2.0.0 requires yiisoft/yii2-composer * -> satisfiable by yiisoft/yii2-composer[2.0.0].
    - Installation request for yiisoft/yii2 2.0.0 -> satisfiable by yiisoft/yii2[2.0.0].
```

How i do?

**Comment by samdark** [comment]
composer self-update

**Comment by EagleMoor** [comment]
```
$ composer self-update
You are already using composer version 1.4.2 (stable channel).
$ composer update
The "extra.asset-installer-paths" option is deprecated, use the "config.fxp-asset.installer-paths" option
```

**Comment by cebe** [comment]
Do you have version of yiisoft/yii2-composer fixed to 2.0.0?
